test/orangeade/generator/term_test.exs

```diff
@@ -0,0 +1,26 @@
+defmodule Orangeade.Generator.TermTest do
+  use ExUnit.Case
+  alias Orangeade.Generator.Term
+
+  test "Check if there are different data types in first 50 generated elements" do
+    list_of_elements = Term.stream()
+    |> Caffeine.Stream.take(100)
+    list_of_conditions = [
+      Enum.any?(list_of_elements, &is_negative/1),
+      Enum.any?(list_of_elements, &is_in_range_32_126/1),
```

Comment:
Why are we testing in the range between 32 and 126.

Reply:
Oh, okay, they're from the character stream. If you call is `ascii_printable?` or `is_ascii_printable?` then the name will convey what it means to be in the range from 32 to 126.